docs/getting-started/welcome.md

```diff
@@ -4,39 +4,38 @@ sidebar_label: Welcome to UMA
+Together, these two components enable the creation of fast, efficient, and secure synthetic derivatives on the Ethereum blockchain.
 
-If you would like to learn more about the concepts behind the DVM and priceless financial contracts, start by reading this [overview](getting-started/overview.md) of the UMA ecosystem. The following sections go into additional detail on priceless DeFi contracts and the DVM.
+## Understanding UMA
+
+If you would like to learn more about the concepts behind UMA's priceless financial contracts and optimistic oracle, start by reading this [overview](getting-started/overview.md) of the UMA ecosystem.
 
 ## For Developers
 
-If you are a developer interested in building with UMA’s technology, start with this [explainer](synthetic-tokens/explainer.md) of how UMA's priceless synthetic tokens work, and then try out this [tutorial](build-walkthrough/mint-locally) for how to deploy your own token on testnet. You can find additional reference materials, including a list of UMA-supported mainnet and testnet deployments, [here](dev-ref/addresses.md).
+If you are a developer interested in building with UMA Protocol, view this [explainer](synthetic-tokens/what-are-synthetic-assets.md) of how UMA's priceless synthetic tokens work and to learn more about UMA's expiring synthetic tokens. 
 
-## For Market Makers
+Additionally, the [development guide](build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA. 
 
-If you are a market maker interested in market making or acting as a liquidator or disputer for financial products deployed with UMA’s priceless synthetic token template, you can find more information about the economic incentives for these roles in this [explainer](synthetic-tokens/explainer.md) for synthetic tokens.
+You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](dev-ref/addresses.md).
 
-## For UMA Token Holders
+## For UMA Tokenholders
 
-If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project token holders.
+If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
 
-## Contracts
+## Contact Us
 
-For a more detailed view, [API documentation](https://docs-dot-uma-protocol.appspot.com/uma/index.html) is available for all UMA smart contracts.
+If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
 
-## Contact Us
 
-If you have any questions or want to reach out, please join our [Discord](https://discord.umaproject.org/).
+
+---
```

Comment:
Did you mean to add this bar at the bottom of the page?

Reply:
No thanks for calling it out. Removed